tests/testthat/test-dm.R

```diff
@@ -144,31 +133,21 @@ test_that("some methods/functions for `zoomed_dm` work", {
 
 test_that("validator is silent", {
   expect_identical(
-    validate_dm(new_dm()),
+    expect_silent(validate_dm(new_dm())),
     empty_dm()
   )
 
   expect_identical(
-    validate_dm(dm_for_filter_w_cycle),
+    expect_silent(validate_dm(dm_for_filter_w_cycle)),
     dm_for_filter_w_cycle
   )
 })
 
-test_that("validator speaks up (postgres)", {
-  skip_if_not("postgres" %in% src_names)
-  expect_dm_error(
-    new_dm3(dm_get_def(dm_for_filter) %>%
-      mutate(data = if_else(table == "t1", list(dm_for_filter_src$postgres$t1), data))) %>%
-      validate_dm(),
-    "dm_invalid"
-  )
-})
-
 test_that("validator speaks up (sqlite)", {
-  skip_if_not("sqlite" %in% src_names)
+  # FIXME: PR #313: will this test fail, if chosen `src` is SQLite?
```

Comment:
Is this test valid only on the database? I'll tweak.

Reply:
we want to test, if the validator can detect it, when tables are on different `src`s.
That's why we need sqlite-version here.
FIXME here is cause I am wondering what happens if the user choses SQLite as test source; in this case all of the tables might be on the same `src`

Reply:
Yes, it will be a different connection.